components/application-operator/charts/application/templates/ingress.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
       }
 spec:
   rules:
-  - host: gateway.{{ .Values.global.domainName }}
+  - host: gateway.{{ .Values.global.applicationConnector.domainName }}
```

Comment:
Those overrides are provided by Application Operator and should not be changed unless you are changing the code which provides the overrides.